### DISCUSSION TITLE
Deprecated gem

### DISCUSSION THREAD

**Discussion by p-brito** [rubygems/rubygems.org#4048] [Q&A] [answered]
Is there any information via API about whether the gem(package) is deprecated?

**Comment by simi** [comment]
Hello, per my understanding there is no gem deprecating feature at rubygems.org and there is no way to mark gem as deprecated at rubygems.org. Do you have any example of gem marked as deprecated at rubygems.org?

**Reply by p-brito** [reply]
Hi @simi, I don't. I was asking because I'm consuming the API to get packages metadata and comparing it to others like Php for example they provide a property saying if it's deprecated or not.

**Reply by simi** [reply, marked as answer]
Then there is no API since there is no given functionality per my understanding built-in rubygems.org.

**Reply by p-brito** [reply]
Thanks @simi
